app/auth/get-redirect-path.js

```diff
@@ -3,9 +3,9 @@ const getRedirectPath = (accessToken, refreshToken, redirect) => {
     redirect = '/landing-page/home'
   }
 
-  const tokenParams = `token=${accessToken}&refreshToken=${refreshToken}`
+  // TODO: refactor as part of access code spike
```

Comment:
Removed the access token from the query string.  We know it's not secure and are going to ditch it.

Likely this function is redundant or needs changing, but can pick that up in the next ticket which is to solve that problem.  So have just simplified it for now.